ui/src/app/tasks-jobs/schedules/schedules.component.html

```diff
@@ -63,14 +69,20 @@ <h1>Schedules</h1>
     <clr-dg-cell>{{ schedule.platform || 'N/A' }}</clr-dg-cell>
     <clr-dg-cell>{{ schedule.cronExpression || 'N/A' }}</clr-dg-cell>
     <clr-dg-action-overflow *ngIf="!grouped">
-      <button class="action-item" (click)="details(schedule)">Details</button>
-      <button class="action-item" (click)="taskDetails(schedule)">Task details</button>
-      <button class="action-item" (click)="createSchedule(schedule)">Add new schedule</button>
-      <button class="action-item" (click)="destroySchedules([schedule])">Destroy</button>
+      <button class="action-item" (click)="details(schedule)">{{ 'schedules.main.details' | translate }}</button>
```

Comment:
Texts like `Name`, `Platform` and `Details` are visibile on other pages.
Can we move these to `commons` translations ?